api/src/main/java/com/codeforcommunity/dto/site/AddSiteRequest.java

```diff
@@ -0,0 +1,182 @@
+package com.codeforcommunity.dto.site;
+
+import com.codeforcommunity.exceptions.HandledException;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+public class AddSiteRequest extends UpdateSiteRequest {
```

Comment:
⛏️ Get and add site request share a lot of common fields, maybe creating an abstract class might help with some of the code duplication?

Reply:
The use cases are very different so they don't abstract super well, eg. the `GET` version has all private final fields and no setters, which doesn't work for the add site request.